README.rst

```diff
@@ -45,6 +55,14 @@ Captures one packet from the capture object.  Can block indefinitely until an ob
 
 Returns the packet data as string (ready for ``decode_ethernet()``), the packet's capture time as a float (with microsecond resolution), and the wire length of the packet.
 
+**cap:next_ex()**
+
+Works and returns the same as cap:next() but preceeded by success value.
+So it returns either of:
+  - true, packet_data, packet_capture_time, packet_length
+  - true, nil -- no packet
+  - false, some_error_message
```

Comment:
The initial success value style is ok for "read-like" functions, where it is important to distinguish between "error" and "no data", but makes the function less useful as an iterator.  I feel this would be better served by modifying the `cap:next()` function to use 'pcap_next_ex()` and call `lua_error()` on failure.

Reply:
OK.